src/fetch/dune.py

```diff
@@ -174,13 +173,13 @@ def get_period_totals(self) -> PeriodTotals:
             realized_fees_eth=int(rec["realized_fees_eth"]),
         )
 
-    def get_period_slippage(self) -> SplitSlippages:
+    def get_period_slippage(self) -> list[DuneRecord]:
```

Comment:
it's a bit inconsistent that some of the methods in the fetcher now return higher level objects (e.g. PeriodTotals above) and some return `list[DunerRecord]`.
Not sure how much work it would be to keep the level of abstraction this class returns consistent (ie return always `list[DuneRecord]` and let the callsite parse)

Reply:
Yea makes sense. I think we can gradually phase out the specific types, but at this moment the others don't need to be touched.